Documentation/Storage-Configuration/Monitoring/ceph-dashboard.md

```diff
@@ -69,6 +69,32 @@ spec:
   dashboard behind a proxy already served using SSL) by setting the `ssl` option
   to be false.
 
+## Visualization of 'Physical Disks' section in the dashboard
+
+Information about physical disks is available only in [Rook host clusters](../../CRDs/Cluster/host-cluster.md).
+
+The Rook manager module is used by the dashboard to obtain the information about physical disks, but it is disabled by default, so displaying the dashboard 'Physical Disks' section will show an error message warning about the situation.
+
+To prepare the Rook manager module to be used in the dashboard, use the following commands in the Ceph toolbox:
+
+```console
+$ ceph mgr module enable rook
```

Comment:
Instead of the toolbox commands, it can actually be enabled with the cluster CR, and rook will take care of setting the backend.
```yaml
  mgr:
    modules:
      - name: rook
        enabled: true
```